rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -251,6 +270,13 @@ public JobId submitFromCatalog(String catalogRestURL, String bucketName, String
         return _getScheduler().submitFromCatalog(catalogRestURL, bucketName, workflowName, variables);
     }
 
+    @Override
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName,
+            Map<String, String> variables, Map<String, String> genericInfos)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        return _getScheduler().submitFromCatalog(catalogRestURL, bucketName, workflowName, variables, genericInfos);
+    }
+
     @Override
     public JobState getJobState(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)